CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
+## Version 2.3.0-beta2
+- Added constructor overloads for TelemetryConfiguration and added creation of a default InMemoryChannel when no channel is specified for a new instance.
+  TelemetryClient will no longer create an InMemoryChannel on the configuration instance if TelemetryChannel is null, instead the configuration instances will always have a channel when created.
+- Added an explicit finalizer to TelemetryConfiguration so that hopefully garbage collection will eventually clean up abondoned ITelemetryChannels which developers did not properly dispose of.
```

Comment:
Good catch, I meant to update the changelog with the information about not disposing of channels that are passed in.